CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I think we should add some badges

Reply:
don't see any badges for it in template

Reply:
I tried to add this comment to the `README.md` but I added into the wrong place.
I realized the `README.md` already include the badges, so you can disregard this message :smile: 

(I'll delete this msg to avoid noise)